tests/wdio/Data/Setup.js

```diff
@@ -0,0 +1,55 @@
+const Page = require('../vaah-webdriverio/Page');
+
+class Setup extends Page{
+    constructor() {
+        super();
+        this.params = {
+            group: {
+                count: null,
+                name: null,
+            }
+        }
+        this.element = {
+            install_button: 'setup-install_vaahcms'
```

Comment:
@we-abhijeet-g001 renama `install_button` to `install_button_testid` so that it's more meaningful